### ISSUE TITLE
SelfControl hangs on "Starting Block" but never finishes

### ISSUE THREAD

**Issue by david24562** [SelfControlApp/selfcontrol#729]
# Summary

 I updated self control from **_Version 3.0.3 (3.0.3)_** to latest **_Version 4.0.1 (408)_** and after installing it does not block anything and is struck since then.

# Background Info

SelfControl Version 4.0.1 (408)
macOS version 10.10.5 
Do you use a VPN/proxy, **NO**
Has this _ever_ worked for you on a prior version of SelfControl? **NO**
Any other special system setup we should know about **NO**

# How Do We Reproduce This Issue?
In macos version 10.10.5
Update your self control app from version 3.0.3 (3.0.3) to 4.0.1 (408) and it will be ask for permission to install a helper tool then type in the password then it will be struck with sign **starting block**

# What _Should_ Have Happened?
I expect self control to block the websites from blocklist

# What Actually Happened Instead?
I updated it to the latest version 4.0.1 (408) and then clicked on start block then it asked for my password to install a helper tool then it was struck and showing no response. Then i closed the self control app and cleared my browsers cache, Restarted the  browser, and then again opened the app and clicked on start block then i gave permission to install a helper tool and again it was struck. I also tried uninstalling and reinstalling the self control but that also did not worked 
![rsz_screen_shot_2021-10-08_at_70256_pm](https://user-images.githubusercontent.com/92163619/136566366-84a28ce2-a4d7-431e-b597-7ecd2d824419.png)

**Comment by suk69** [comment]
The same happens with me too....even I've opened a query but someone responded once and hasn't replied since then.

**Comment by david24562** [comment]
is nobody going to help???????

**Comment by david24562** [comment]
thank you very much **@cstigler** for the response but unfortunately it is still struck. I sat for 30 mins but it did not start and for the meantime i will be using the older version(3.0.3) which is working on my macbook pro.

@cstigler is it possible if self control app is not working because of my macos version or macbook hardware

**Comment by cstigler** [comment]
Huh @david24562, that's a bummer and unfortunately I'm not sure what the issue is! It's totally possible it's related to your macOS version or hardware, since the macOS version is very old, but I'm not sure. I'd try upgrading your Mac to Big Sur or Monterey if possible.

I'll leave this open but probably won't have a chance to debug it in the near future - I think I've got at least a 6-month backlog of issues to look at based on my current pace 🙈. I hope 3.0.3 continues to work for now!

**Comment by david24562** [comment]
yeah @cstigler 3.0.3 is working for me. I understand the problem with backlog, so best of luck debugging the issues. And as per upgrading my macbook the compatible operating system available is macOS Catalina 10.15.7.I will try to upgrade my macbook to Catalina and hardware too
